src/components/ButtonsGroup/__tests__/ButtonsGroup.test.tsx

```diff
@@ -25,46 +26,60 @@ describe('ButtonsGroup', () => {
   });
 
   it('should renders correctly with default props', () => {
-    const { getByA11yLabel } = render(<ButtonsGroup {...initialProps} />);
+    const { getAllByTestId } = render(<ButtonsGroup {...initialProps} />);
 
-    expect(getByA11yLabel('lorem')).toHaveStyle({
+    expect(getAllByTestId('Button')[0]).toHaveStyle({
       backgroundColor: colors.solidPrimaryMedium,
       borderRadius: radius.small,
     });
 
-    expect(getByA11yLabel('ipsum')).toHaveStyle({
+    expect(getAllByTestId('Button')[1]).toHaveStyle({
       backgroundColor: colors.transparentFaintSemitransparent,
       borderRadius: radius.small,
     });
   });
 
   it('should renders correctly with inversed prop', () => {
-    const { getByA11yLabel } = render(<ButtonsGroup {...initialProps} inversed />);
+    const { getAllByTestId } = render(<ButtonsGroup {...initialProps} inversed />);
 
-    expect(getByA11yLabel('ipsum')).toHaveStyle({
+    expect(getAllByTestId('Button')[1]).toHaveStyle({
       backgroundColor: colors.transparentBrightSemitransparent,
       borderRadius: radius.small,
     });
   });
 
-  it('should renders correctly with rounded prop', () => {
-    const { getByA11yLabel } = render(<ButtonsGroup {...initialProps} rounded />);
+  it('should renders correctly with rounded when prop', () => {
```

Comment:
I think there is a typo here
```suggestion
  it('should renders correctly with rounded prop', () => {
```

Reply:
Fixed: a007ede58c5858c65941aee4d62b19e98363f280